packages/babel-generator/src/generators/jsx.js

```diff
@@ -1,8 +1,9 @@
 export function JSXAttribute(node: Object) {
   this.print(node.name, node);
   if (node.value) {
+    const raw = this.getPossibleRaw(node);
     this.token("=");
-    this.print(node.value, node);
+    this.print(raw || node.value, node);
```

Comment:
I don't think this particular call is actually doing anything, because `JSXAttribute` nodes don't have a `.raw`, their `.value` property does. I'm assuming this is working entirely because `StringLiteral` already checks for `.raw`.